src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ CHANGELOG
  * Added php ini session options `sid_length` and `sid_bits_per_character` 
    to the `session` section of the configuration
  * Added support for Translator paths, Twig paths in translation commands.
+ * Added `SuggestMissingPackageSubscriber` for suggesting missing packages when using console commands
```

Comment:
not sure we need  to advertise this internal class =/

Reply:
done